testing/regress/hpcc/regression/suite.py

```diff
@@ -126,8 +126,17 @@ def addFileToSuite(self, eclfile):
             versions = eclfile.getVersions()
             versionId = 1
             for version in versions:
-                files.append({'basename':basename, 'version':version,  'id':versionId })
-                versionId += 1
+                if 'no'+self.name in version:
+                    # Exclude it from this target
+                    pass
+                else:
+                    # Remove exclusion key(s) from version string
+                    config = getConfig();
```

Comment:
The lastes version of engine should display an error message for the second attempt:
./ecl-test query test.ecl -t=hthor2
if the hthor2 doesn't exists in Clusters array of config file and halt. (It does it for me if I try to run that command on my system)
If you have this cluster in your config, then you should use //nohthor2 in your file to exclude.

Or if you want to //nothor tag exclude the file from all clusters which name starts with hthor then I think, it is a different issue.